src/main/webapp/i18n/en/error.json

```diff
@@ -45,6 +45,7 @@
         "submissionPolicyAlreadyDisabled": "The submission policy could not be disabled, because it is already inactive.",
         "submissionPolicyToggleFailedPolicyNotExist": "The submission policy could not be toggled, because the programming exercise does not have a submission policy.",
         "submissionPolicyActiveNull": "Submission policies must be activated or deactivated. Activation cannot be null.",
-        "submissionPolicyUpdateFailedPolicyNotExist": "The submission policy could not be updated, because the programming exercise does not have a submission policy."
+        "submissionPolicyUpdateFailedPolicyNotExist": "The submission policy could not be updated, because the programming exercise does not have a submission policy.",
+        "oneActivePlagiarismCheck": "There is already a plagiarism check in this course!"
```

Comment:
Add something like "Please wait until the current plagiarism check has finished until you start another one"